src/goats_cli/cli.py

```diff
@@ -15,6 +14,55 @@
 from goats_cli.exceptions import GOATSClickException
 from goats_cli.modify_settings import modify_settings
 from goats_cli.process_manager import ProcessManager
+from goats_cli.versioning import VersionChecker
+
+UPDATE_DOC_URL = "https://goats.readthedocs.io/en/stable/update.html"
+
+
+def _check_version() -> None:
+    """
+    Check whether GOATS is outdated.
+
+    Returns
+    -------
+    None
+        This function does not return a value. It may block for user input and
+        can raise ``click.Abort`` if the user cancels
```

Comment:
Remove this `Returns` and move the helpful message into the above docstring summary since this returns `None`.